packages/schema-blocks/src/functions/presenters/SidebarWarningPresenter.ts

```diff
@@ -2,8 +2,8 @@ import { getBlockType } from "../BlockHelper";
 import { select } from "@wordpress/data";
 import { __ } from "@wordpress/i18n";
 import { get } from "lodash";
-import { BlockValidationResult } from "../../core/validation";
-import { BlockValidation } from "../../core/validation";
+import { BlockValidation, BlockValidationResult } from "../../core/validation";
+import { getAllDescendantIssues } from "../validators/getAllDescendantIssues";
```

Comment:
mooier om in `validators/index.ts` deze functie te importeren en exporteren, zodat je hier
```suggestion
import { getAllDescendantIssues } from "../validators";
```
kunt doen

Reply:
Done